sale_stock_prebook/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2023 Michael Tietz (MT Software) <mtietz@mt-software.de>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+{
+    "name": "Sale Stock Prebook",
+    "summary": "Add process to prebook a sale order's stock before confirming it",
+    "version": "16.0.1.0.0",
+    "author": "MT Software, BCIM, ACSONE SA/NV, Odoo Community Association (OCA)",
```

Comment:
```suggestion
    "author": "MT Software, BCIM, Odoo Community Association (OCA)",
```
Sorry but I think your changes do not entitle you to add your company as an author. I tried to do this in several other PRs, with a lot of more code changes. And it was always blocked. 
@jbaudoux @lmignon is there any guidance from the oca how to handle this?

Reply:
I'm not going to fight. I have removed acsone from the list of authors.